src/GitHubCreateMergePRs/Program.cs

```diff
@@ -157,4 +156,53 @@ private static async Task RunAsync(XDocument config, bool isAutomatedRun, bool i
             }
         }
     }
+
+    private static bool ShouldRunMerge(XElement merge, bool isAutomatedRun)
```

Comment:
should we doc what frequencies are supported?
as far as I can tell it's daily and weekly, but perhaps we should make it obvious.